x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/export/get_export_rule_action_connectors.ts

```diff
@@ -42,11 +43,30 @@ const mapExportedActionConnectorsDetailsToDefault = (
     excluded_action_connections: exportDetails.excludedObjects,
   };
 };
+const filterOutPredefinedActionConnectorsIds = async (
+  actionsClient: ActionsClient,
+  actionsIdsToExport: string[]
+): Promise<string[]> => {
+  const allActions = await actionsClient.getAll();
```

Comment:
It would be great if we could pass a boolean to `actionsClient.getAll()` that would allow us to skip adding the predefined actions to the response. Looks like the `getAll` function appends the predefined connectors after fetching the action saved objects. 

https://github.com/elastic/kibana/blob/686ccad2183ec2fc36a70d2d9f20d28f270db247/x-pack/plugins/actions/server/actions_client.ts#L444


I'm wondering if it is simpler to just push this logic down to the api? What do you think?

Reply:
Thank you, Devin, for looking into it :) 

I need to get the `Preconfigured Connectors` in the response because there might be some in the `actionsIdsToExport` and I want to filter out those from the `actionsIdsToExport`